services/web-server/src/graphql/Notify.graphql

```diff
@@ -0,0 +1,28 @@
+type NotificationAddress {
+  notificationType: String!
```

Comment:
Instead of `String`, let's make this one of `email`, `pulse`, `irc-user`, `irc-channel` using `enum`.